tests/Components/test_FreeBody.py

```diff
@@ -20,22 +21,30 @@ def freebody():
 
 
 def test_instantiation(freebody):
-    assert(freebody.position_cg.x == -0.2)
-    assert(freebody.position_cg.y == 0.02)
-    assert(freebody.pitch_rot_inertia == 5.2)
-    assert(freebody.ambient.temperature == 273.15)
+    assert freebody.position_cg.x == -0.2
+    assert freebody.position_cg.y == 0.02
+    assert freebody.pitch_rot_inertia == 5.2
+    assert freebody.ambient.temperature == 273.15
 
 
 def test_states(freebody):
     freebody.position = Vector2(15.1, 1.2)
     freebody.angle = math.radians(15)
     freebody.velocity = Vector2(12.1, 0.7)
     freebody.rot_velocity = 14
-    assert(freebody.position == Vector2(15.1, 1.2))
-    assert(freebody.angle == math.radians(15))
-    assert(freebody.velocity == Vector2(12.1, 0.7))
-    assert(freebody.rot_velocity == 14)
+    assert freebody.position == Vector2(15.1, 1.2)
+    assert freebody.angle == math.radians(15)
+    assert freebody.velocity == Vector2(12.1, 0.7)
+    assert freebody.rot_velocity == 14
 
 
 def test_gravitational_center(freebody):
-    assert(freebody.gravitational_center == Vector2(-0.2, 0.02))
+    assert freebody.gravitational_center == Vector2(-0.2, 0.02)
+
+
+def test_get_total_weight(freebody):
+    freebody.angle = math.radians(15)
+    weight, weight_point = freebody.get_total_weight()
+    npt.assert_almost_equal(weight.r, 229.47, decimal=2)
+    npt.assert_almost_equal(weight.theta, -1.83, decimal=2)
```

Comment:
It would be nice to have this weight.theta asserted in degrees, like so:
```
npt.assert_almost_equal(weight.theta, math.radians("angulo"), decimal=2)
```

It gets easier for the developers to maintain.